config/read_write_database_sample.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
Yes can be removed, I didn't understand at first it was the same syntax as the laravel docs.